postman/collections/Interledger/Rafiki Admin APIs/Create Asset.bru

```diff
@@ -0,0 +1,46 @@
+meta {
+  name: Create Asset
+  type: graphql
+  seq: 1
+}
+
+post {
+  url: {{RafikiGraphqlHost}}/graphql
+  body: graphql
+  auth: none
+}
+
+body:graphql {
+  mutation CreateAsset($input: CreateAssetInput!) {
+    createAsset(input: $input) {
+      asset {
+        code
+        createdAt
+        id
+        scale
+        withdrawalThreshold
+        liquidityThreshold
+      }
+      code
+      message
+      success
+    }
+  }
+}
+
+body:graphql:vars {
+  {
+    "input": {
+      "code": "USD",
+      "scale": 6,
+      "withdrawalThreshold": null,
+      "liquidityThreshold": "100000000"
+    }
+  }
+}
+
+script:post-response {
+  const body = res.getBody();
+  
+  bru.setEnvVar("assetId", body.data.createAsset.asset.id);
```

Comment:
```suggestion
  bru.setEnvVar("assetId", body.data.createAsset.asset?.id);
```
and throughout, so we can get the proper error out if it occurs:
![Screenshot 2024-02-19 at 18 17 00](https://github.com/interledger/rafiki/assets/15069181/eb92b50c-2902-463e-87d0-2b6122c8aaa0)
![Screenshot 2024-02-19 at 18 16 47](https://github.com/interledger/rafiki/assets/15069181/2e73317e-092e-47ac-8eb2-d92a1a1571ad)